_posts/2023-02-07-Announcing-Data-Prepper-2.1.0.md

```diff
@@ -0,0 +1,253 @@
+(`durationInNanos`) could detect abnormally high latencies. The following anomaly detector processor configuration 
+can be used for it.
+
+```
+trace-metric-anomaly-detector-pipeline:
+  source:
+    pipeline:
+      name: "trace-high-latency-metrics-pipeline"
+  processor:
+    - anomaly_detector:
+        keys: ["max"]
+        mode:
+          random_cut_forest:
+  sink:
+    - opensearch
+         hosts: ["https://opensearch:9200"]
+         insecure: true
+         username: "admin"
+         password: "admin"
+         index: high-latency-traces-with-anomalies
+```
+
+## Sampling and limiting
+
+Data Prepper 2.1 supports sampling and rate-limiting to limit the number of events that are sent to a Sink.
+
+One possible use case for both these features can be used to reduce the load on the OpenSearch when storing normal 
+(not very interesting) logs and metrics.
+
+Both these features are available as a configurable action in the aggregate processor. Simply choose the appropriate 
+action (sampling or rate limiting) and specify the sampling percent or events per second option as needed.
+
+For example, the following percent sampler configuration sends 60% of the events to OpenSearch
+
+```
+trace-normal-pipeline:
+  source:
+    pipeline:
+      name: "span-pipeline"
+  processor:
+    - aggregate:
+        identification_keys: ["serviceName"]
+        action:
+          percent_sampler:
+            percent: 60
+        group_duration: "30s"
+        aggregate_when: "/status_code != 2 and /durationInNanos <= 1000000000"
+ sink:
+    - opensearch
+         hosts: ["https://opensearch:9200"]
+         insecure: true
+         username: "admin"
+         password: "admin"
+         index: sampled-traces
+```
+
+Rate limiter action can be configured to limit the number of events by specifying the number of events per second and 
+optionally what to do when the number of events exceed the specified limit using the `when_exceeds` option. This 
+`when_exceeds` option can take either `block` or `drop` value and can either block until events are allowed or drop any 
+excess events. The `block` is useful if there is a temporary burst and the user do not want to lose any events in that 
+case. The default value for this option is `block`.
+
+And the following rate limiter configuration send 10 events per second to the open search and using `drop` option.
+
+```
+trace-normal-pipeline:
+  source:
+    pipeline:
+      name: "span-pipeline"
+  processor:
+    - aggregate:
+        identification_keys: ["serviceName"]
+        action:
+          rate_limiter:
+            events_per_second: 10
+            when_exceeds: "drop"
+        group_duration: "30s"
+        aggregate_when: "/status_code != 2 and /durationInNanos <= 1000000000"
+ sink:
+    - opensearch
+         hosts: ["https://opensearch:9200"]
+         insecure: true
+         username: "admin"
+         password: "admin"
+         index: sampled-traces
+```
+
+## OTel logs
```

Comment:
```suggestion
## Otel logs
```

Reply:
This is an abbreviation for OpenTelemetry which is a proper noun.

> OpenTelemetry, also known as OTel for short,

https://opentelemetry.io/docs/

Reply:
In that case, this should read as: "OpenTelemetry (OTel) logs".

Reply:
Since we're not using the abbreviation in the section body, does it make sense to leave it in the spelled-out form? "OpenTelemetry logs"?